### PR TITLE
check if backup has been completed

### LINKED ISSUE #4483: Sweep paper wallet doesn't check if backup has been completed

### PR DESCRIPTION
**Pull request by gabrielbazan7**
fixes #4483

### CODE REVIEW

File: public/views/paperWallet.html [hunk 1]
```diff
@@ -5,61 +5,78 @@
...
+  <div class="row" ng-show="index.needsBackup">
+    <div class="columns">
+      <h4></h4>
+      <div class="size-14 text-warning m20b">
+        <i class="fi-alert size-12"></i>
+        <span class="text-warning" translate>Backup Needed</span>.
+        <span translate>Before receiving funds, it is necessary backup your wallet. If you lose this device, it is impossible to access your funds without a backup.</span>
+      </div>
+      <div class="text-center m20t">
+      <button class="button outline round dark-gray" href ui-sref="preferences" >
```
Comment (cmgustavo) [inline public/views/paperWallet.html]: Should be `<a href ui-sfref...>`, not button.

File: public/views/paperWallet.html [hunk 2]
```diff
@@ -5,61 +5,78 @@
...
-  <h4 ng-show="!error"></h4>
-  <div class="box-notification m20b" ng-show="error">
-    <span class="text-warning">{{error|translate}}</span>
+  <div class="row" ng-show="index.needsBackup">
+    <div class="columns">
+      <h4></h4>
+      <div class="size-14 text-warning m20b">
+        <i class="fi-alert size-12"></i>
+        <span class="text-warning" translate>Backup Needed</span>.
+        <span translate>Before receiving funds, it is necessary backup your wallet. If you lose this device, it is impossible to access your funds without a backup.</span>
```
Comment (isocolsky) [inline public/views/paperWallet.html]: > Before receiving funds, it is necessary backup your wallet.

Should be "Before receiving funds, it is necessary **to** backup your wallet."

**Comment by gabrielbazan7** [comment]
Updated

**Comment by cmgustavo** [comment]
ACK.

**Pull request merged**